fpm/build-deb

```diff
@@ -56,4 +54,7 @@ fpm -t deb \
     --description "Soundnode App is an Open-Source project to support Soundcloud for desktop Mac, Windows, and Linux." \
     --after-install "fpm/create-symlink" \
     --after-remove "fpm/remove-symlink" \
-    --deb-no-default-config-files
+    --deb-no-default-config-files \
+    "$DIST_DIR/.=/usr/share/soundnode/" \
+    "app/soundnode.png=/usr/share/pixmaps/" \
```

Comment:
Typically the "icons" folder contains icons related to the Gnome (and KDE I assume, but I have no knowledge of this) themes. The "pixmaps" folder contains icons related to specific applications.